### ISSUE TITLE
Is there documentation for SQLAlchemySchema's or SQLAlchemyAutoSchema's `load` method?

### ISSUE THREAD

**Issue by alichaudry** [marshmallow-code/marshmallow-sqlalchemy#619]
I've looked at the documentation [here](https://marshmallow-sqlalchemy.readthedocs.io/en/latest/api_reference.html#marshmallow_sqlalchemy.SQLAlchemySchemaOpts) and under `sqla_session` it says `You can also pass a session to the Schema’s load method.` However, it's unclear how to pass the session to the `load` method. Is it `sqla_session` as well, or something else? I've searched the documentation for the signature of this method but I was unable to find any details.

I _was_ able to make it work by looking at the function signature in the source code [here](https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/dev/src/marshmallow_sqlalchemy/load_instance_mixin.py#L88).

Is this just a case of missing documentation, or am I using an undocumented feature by having to look in the source code?

**Comment by sloria** [comment]
Good catch! It's documented [here](https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/91d83123b7b21076d238cf079f5678b78ca176b5/src/marshmallow_sqlalchemy/load_instance_mixin.py#L91), but I don't think this appears in the sphinx docs. It is public API, so we should definitely document this more clearly
